lib/smarter_csv/hash_transformations.rb

```diff
@@ -42,6 +42,8 @@ def self.convert_values_to_numeric(hash, args=nil)
         case hash[k]
         when /^[+-]?\d+\.\d+$/
           hash[k] = hash[k].to_f
+        when /^[+-]?\d+\,\d+$/
+          hash[k] = hash[k].gsub!(',', '.').to_f
```

Comment:
actually this will not always work... in Europe decimal numbers are 3000,5
but in the US, they might use the comma as a separator for thousand, e.g.: 3,000.5
in Europe they use the dot as the separator for thousand: 3.000,5

all the same number ... so in this case, it's better to modify the value with a custom hash_transformation

This should not be default behavior

Reply:
@vincenzodev please check this page: https://github.com/tilo/smarter_csv/wiki/Parsing-Floats,-Integers,-Money,-and-Dates